src/mono/wasm/testassets/BlazorHostedApp/BlazorHosted.Server/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Configuration;
+using System;
+using Microsoft.Extensions.Logging;
+using BlazorHosted.Server.Hubs;
+
+var builder = WebApplication.CreateBuilder(args);
+
+ // Add command-line arguments to configuration to pass wbt LogRootPath that is availabe in Azure
+var configuration = new ConfigurationBuilder()
+    .AddCommandLine(args)
+    .Build();
+
+var logRootPath = configuration["logRootPath"] ?? "logs";
+
+// Add file logging
+builder.Logging.AddFile(logRootPath + "/myapp-{Date}.txt");
```

Comment:
Isn't logging to console enough? The logs are picked up by playwright integration

Reply:
Yes, it is but only the client side. It's for scenario when signalR is able to connect to the server and send the message but the server cannot reply back. The exceptions from the server won't get logged in playwright

Reply:
Maybe I'll reply with the hint I got from @maraf:
we have server side logging in https://github.com/dotnet/runtime/blob/d4643138e8b9f23482618a32535cc40552da0bb7/src/mono/wasm/Wasm.Build.Tests/BrowserRunner.cs#L47, we just don't surface it. It should be fixed, then we can remove file logging.